src/io/openshift/plugins/analytics.groovy

```diff
@@ -1,10 +1,29 @@
 package io.openshift.plugins
 
 import io.openshift.Events
+import io.openshift.Utils
 
 def register() {
     Events.on("build.pass") {
-        e, a -> echo "invoking bayesian analytics $e $a"
+        e, a -> echo "invoking bayesian analytics"
+        def image = config.runtime()
+        if(!image) {
+            return
+        }
+        try {
+            def scmUrl = scm.getUserRemoteConfigs()[0].getUrl()
+            echo "YYYYYYYUUUUYYSYYSYS ${scmUrl}"
```

Comment:
@yzainee What is this? You really need to review your own code before you push to GH. This should be mandatory.

Reply:
@pradeepto Please refer to the comment from @hrishin where he wanted to try out the scm stuff.. i will anyway roll back this change.

FYI, scm gives RejectedAccessException @sthaha